platform/output.go

```diff
@@ -239,3 +240,37 @@ func printLines(content string, contextLine int, line int, skipHighlight bool) {
 	}
 	fmt.Printf("%s%s\n", tableDown, strings.Repeat(tableSep, getTerminalWidth()-noLineWidth-1))
 }
+
+func printSarifProblem(r *sarif.Result, ruleId, message string) {
+	if r.Locations[0].PhysicalLocation != nil {
+		printProblem(
+			ruleId,
+			getSeverity(r),
+			message,
+			r.Locations[0].PhysicalLocation.ArtifactLocation.Uri,
+			int(r.Locations[0].PhysicalLocation.Region.StartLine),
+			int(r.Locations[0].PhysicalLocation.Region.StartColumn),
+			int(r.Locations[0].PhysicalLocation.ContextRegion.StartLine),
+			r.Locations[0].PhysicalLocation.ContextRegion.Snippet.Text,
+		)
+	} else {
+		printProblem(
+			ruleId,
+			getSeverity(r),
+			message,
+			"",
+			0,
+			0,
+			0,
+			"",
+		)
+	}
+}
+
+// getProblemsFoundMessage returns a message about the number of problems found, used in CLI and BitBucket report.
+func getProblemsFoundMessage(newProblems int) string {
+	if newProblems == 0 {
+		return "It seems all right 👌 No new problems found according to the checks applied"
+	}
+	return fmt.Sprintf("Found %d new problems according to the checks applied", newProblems)
```

Comment:
Oh, never noticed that the message is wrong for exactly 1 result 😅 So, very minor, but let's fix that :D